packages/react-components/react-theme/src/alias/darkColor.ts

```diff
@@ -37,11 +37,12 @@ export const generateColorTokens = (brand: BrandVariants): ColorTokens => ({
   colorBrandForeground1: brand[100], // #2899f5 Global.Color.Brand.100
   colorBrandForeground2: brand[110], // #3aa0f3 Global.Color.Brand.110
   colorNeutralForeground1Static: grey[14], // #242424 Global.Color.Grey.14
-  colorNeutralForegroundInverted: white, // #ffffff Global.Color.White
-  colorNeutralForegroundInvertedHover: white, // #ffffff Global.Color.White
-  colorNeutralForegroundInvertedPressed: white, // #ffffff Global.Color.White
-  colorNeutralForegroundInvertedSelected: white, // #ffffff Global.Color.White
+  colorNeutralForegroundInverted: grey[14], // #242424 Global.Color.Grey.14
```

Comment:
The alias files are supposed to be generated by a token pipeline. Have you updated the token pipeline or edited these files manually?

Reply:
I've edited them manually. What are the steps for updating via the pipeline?

Reply:
Talked offline and we don't want the pipeline work to block this PR so we'll go with the manual changes. Pipeline work will be done ASAP.